test/ParameterStoreBuildWrapperPluginTest.groovy

```diff
@@ -25,6 +26,46 @@ class ParameterStoreBuildWrapperPluginTest {
             Collection actualPlugins = TerraformEnvironmentStage.getPlugins()
             assertThat(actualPlugins, hasItem(instanceOf(ParameterStoreBuildWrapperPlugin.class)))
         }
+
+        @Test
+        void modifiesTerraformEnvironmentStageCommandWithGlobalParameter() {
+            ParameterStoreBuildWrapperPlugin.withGlobalParameter('/somePath/').init()
+
+            Collection actualPlugins = TerraformEnvironmentStage.getPlugins()
+            assertThat(actualPlugins, hasItem(instanceOf(ParameterStoreBuildWrapperPlugin.class)))
+        }
+
+        @Test
+        void modifiesTerraformEnvironmentStageCommandWithGlobalParameterAndOptions() {
+            ParameterStoreBuildWrapperPlugin.withGlobalParameter('/somePath/', [someKey: true, anotherKey: 'someValue']).init()
+
+            Collection actualPlugins = TerraformEnvironmentStage.getPlugins()
+            assertThat(actualPlugins, hasItem(instanceOf(ParameterStoreBuildWrapperPlugin.class)))
+        }
```

Comment:
Same.  We're unit testing `init()`, which does not use `withGlobalParameter`.  I would move or remove this test.

Reply:
removed